R/ttestindependentsamples.R

```diff
@@ -429,11 +429,11 @@ ttestIndependentMainTableRow <- function(variable, dataset, test, testStat, effS
 .ttestIndependentEqVarRow <- function(table, variable, groups, dataset) {
   levene <- car::leveneTest(dataset[[ .v(variable) ]], dataset[[ .v(groups) ]], "mean")
 
-  F  <- levene[1, "F value"]
-  df <- levene[1, "Df"]
+  t  <- sqrt(levene[1, "F value"])
```

Comment:
If you modify a function and encounter `.v()` (or `.unv()`), please remove it!

```r
levene <- car::leveneTest(dataset[[variable]], dataset[[groups]], "mean")
```

Reply:
ah yes, done. I thought id just leave it how i found it, but now i know!